wpimath/src/main/native/include/frc/kinematics/ChassisSpeeds.h

```diff
@@ -38,6 +38,24 @@ struct WPILIB_DLLEXPORT ChassisSpeeds {
    */
   units::radians_per_second_t omega = 0_rad_per_s;
 
+  /**
+   * Creates a new ChassisSpeeds object.
+   *
+   * @param vx The desired speed along the X axis.
+   * @param vy The desired speed along the Y axis.
+   * @param omega The desired rotational velocity.
+   */
+  constexpr ChassisSpeeds(units::meters_per_second_t vx,
+                          units::meters_per_second_t vy,
+                          units::radians_per_second_t omega)
+      : vx{vx}, vy{vy}, omega{omega} {};
+
+  constexpr ChassisSpeeds() = default;
```

Comment:
Should we be defining constructors at all? Currently this struct can be initialized with a [designated initializer list](https://en.cppreference.com/w/cpp/language/aggregate_initialization).

Here's an example test which passes on `main` but fails to compile on this PR:

```cpp
TEST(ChassisSpeedsTest, Initialize) {
  constexpr frc::ChassisSpeeds speeds{.vx = 2_mps};

  EXPECT_NEAR(2.0, speeds.vx.value(), kEpsilon);
  EXPECT_NEAR(0.0, speeds.vy.value(), kEpsilon);
  EXPECT_NEAR(0.0, speeds.omega.value(), kEpsilon);
}
```

Reply:
Yeah I think this is good point, I'll remove them